corehq/apps/toggle_ui/models.py

```diff
@@ -1 +1,68 @@
-# This file is only here so that django will recognize that this is a valid app
+from django.db import models
+
+from corehq.toggles import NAMESPACE_USER, NAMESPACE_DOMAIN, NAMESPACE_OTHER
+
+
+class ToggleAuditManager(models.Manager):
```

Comment:
just for my own education - what's the reason for implementing this via a manager (versus e.g. directly on the model)?